runtests.sh

```diff
@@ -327,7 +327,8 @@ export PYTHONPATH="$homedir:$PYTHONPATH"
 echo "PYTHONPATH: $PYTHONPATH"
 
 # by default do nothing
-cmdPrefix=""
+# cmdPrefix=""
+cmdPrefix="$(PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python) "  # FIXME https://github.com/Project-MONAI/MONAI/issues/4354
```

Comment:
don't any idea how to set the environment variable before running cmd? this doesn't seem to work @rijobro @ericspod

Reply:
You're resolved it with the `export` which is what I'd have suggested.